migrations/20200715014355-create-location-pins.js

```diff
@@ -0,0 +1,40 @@
+"use strict";
+
+module.exports = {
+  up: (queryInterface, Sequelize) => {
+    return queryInterface.createTable("locationPins", {
```

Comment:
since this is a new table we're adding, i'd like to try out paranoid tables - it basically implements soft deletes which we want but aren't doing with any existing table currently
https://sequelize.org/master/manual/paranoid.html